examples/src/main/scala/za/co/absa/atum/examples/CreateInfoFileToolCSV.scala

```diff
@@ -74,13 +77,25 @@ object CreateInfoFileToolCSV {
       .option("header", haveHeaders)
       .load(inputFileName)
 
-    val strJson = ControlMeasureUtils.createInfoFile(ds,
-      sourceApplication,
-      inputFileName,
-      dateInDMYFormat,
-      infoVersion,
-      historyType = historyType,
-      country = country,
-      aggregateColumns = columnNames.toSeq)
+    val cm: ControlMeasure = ControlMeasureBuilder
+      .forDF(ds)
+      .withAggregateColumns(columnNames.toSeq)
+      .withSourceApplication(sourceApplication)
+      .withInputPath(inputFileName)
+      .withReportDate(dateInDMYFormat)
+      .withReportVersion(infoVersion)
+      .withCountry(country)
+      .withHistoryType(historyType)
+      .withSourceType("Source")
+      .withInitialCheckpointName("Source")
+      .withWorkflowName("Source")
+      .build
+
+    val hadoopConfiguration = ds.sparkSession.sparkContext.hadoopConfiguration
+    val (fs, path) = InfoFile.convertFullPathToFsAndRelativePath(inputFileName)(hadoopConfiguration)
+
+    ControlMeasureUtils.writeControlMeasureInfoFileToHadoopFs(cm, path, JsonType.Pretty)(fs)
+
+    val strJson = cm.asJsonPretty
```

Comment:
Similar as above, I would put this together with the actual `cm` creation (to line 94).